Release notes for Red Hat Developer Hub 1.6.0 (#1133)
redhat-developer/red-hat-developers-documentation-rhdh: modules/release-notes/ref-release-notes-fixed-issues.adoc

[id="fixed-issues"]
= Fixed issues

This section lists issues fixed in {product} {product-version} that have a significant impact on users.

== Fixed issues in 1.6.0

[id="bug-fix-rhidp-6042"]

> === Floating Action Button (FAB) positioned in the &#39;Bottom-Left&#39; slot on {product-short}

Reviewer: I'm not sure why there are unexpected characters appearing in the section instead of the single quotes —could it be related to the script? @jmagak 
```suggestion
=== Floating Action Button (FAB) positioned in the bottom-left on {product-short}

```

Author: > I'm not sure why there are unexpected characters appearing in the section instead of the single quotes —could it be related to the script? @jmagak

Yes, Heena. These are related to the script. The preview is rendered correctly though.

> Previously, the Floating Action Button (FAB) appeared over the navigation sidebar when the slot was set to &#39;bottom-left&#39;. This placement obstructed access to navigation elements, potentially hindering user interaction.

Reviewer: ```suggestion
Previously, the Floating Action Button (FAB) appeared over the navigation sidebar when the slot was set to bottom-left. This placement obstructed access to navigation elements, potentially hindering user interaction.

```

> With this update, the FAB&#39;s position is adjusted to render adjacent to the navigation for the &#39;bottom-left&#39; slot position. As a result, users can access navigation options without obstruction.

Reviewer: ```suggestion
With this update, the FAB's position is adjusted to render adjacent to the navigation for the bottom-left slot position. As a result, users can access navigation options without obstruction.
```

.Additional resources
* link:https://issues.redhat.com/browse/RHIDP-6042[RHIDP-6042]

[id="bug-fix-rhidp-6448"]
=== Manually added resolutions override resolutions added by `--suppress-native-package`

> Previously, manually added resolutions were not overwritten by the package export-dynamic-plugin command. This could lead to incorrect package dependencies in the exported dynamic plugin.

Reviewer: ```suggestion
Earlier, the `export-dynamic-plugin` command did not overwrite manually added resolutions, which could result in incorrect package dependencies in the exported dynamic plugin.

```

[id="fixed-issues"]
= Fixed issues

> This section lists issues fixed in {product} {product-version} that have a significant impact on users.

Reviewer 2: the suffix "that have a significant impact on users." was manually removed in the docs for 1.5, but the release notes scripted template was NOT updated (so it's re-appearing here). 

See PRs to remove the same thing for 1.6+ 
* https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/1143 (1.6)
* https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/1145 (1.5)
* https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/1146 (main)
